source/api/image/2.1/change-log-211.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Image API 2.1.1 Change Log"
+title_override: "Changes for IIIF Image API Version 2.1.1"
+id: image-api-211-change-log
+layout: spec
+tags: [specifications, image-api, change-log]
+major: 2
+minor: 1
+patch: 1
+pre: final
+redirect_from:
+  - /api/image/2.1/change-log-211.html
+---
+
+This document is a companion to the [IIIF Image API Specification, Version 2.1.1][image-api]. It describes the editorial changes to the API made in this patch release. The changes are all clarifications, typo corrections or to related documents that are not [semantically versioned][semver] such as example resources and resources to manage transformation to and from the JSON-LD serialization.
```

Comment:
> such as example resources and resources to manage transformation to and from the JSON-LD serialization.

...such as inline examples and resources to manage transformation to and from the JSON-LD serialization.